lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1008,7 +1027,8 @@ group_config(rsa = Type, Config) ->
                   rsa_oaep(),
                   no_padding()
                  ],
-    [{sign_verify, SignVerify}, {pub_priv_encrypt, PubPrivEnc} | Config];
+    Generate = [{rsa, 1024, 3}, {rsa, 2048, 17}, {rsa, 3072, 65537}],
```

Comment:
Ahh. Chasing down some references, it looks like the required minimum key length for FIPS 140-2 validation is currently 2048 bits ([from "FCS_CKM.1.1(1)" here](https://www.niap-ccevs.org/pp/pp_app_v1.2-table.htm)).

There's probably no need to test key generation with a huge number of parameters; I'll just remove the 1024-bit test case.